packages/core/src/framework/reducers/reducer.class.ts

```diff
@@ -1,6 +1,6 @@
 import { DeepState, DependencyResults, DependencyStates, ShallowState } from './state';
 
-import { AuguryEvent, ElapsedAuguryEvent } from '../events';
+import { AuguryEvent } from '../events';
 import { merge, objToPairs, shallowClone } from '../utils';
 
```

Comment:
I made a story for more reducer refactoring. the code here is busy and needs some more